packages/internal/src/index.js

```diff
@@ -7,6 +7,8 @@ export * from './debug.js';
 export * from './utils.js';
 export * from './method-tools.js';
 export * from './typeGuards.js';
+// eslint-disable-next-line import/no-unresolved -- implemented as .d.ts
+export * from './types.js';
```

Comment:
This causes many errors.  `types.js` must exist, even if you are only using the import to obtain types from `types.d.ts`.